### PR TITLE
Art.documentation fixes

### PR DESCRIPTION
**Pull request by ATeal**
## Description of change
Documentation updates to support Swagger curl generation

## Testing done
local calls from the dev portal

#### Unique to this PR

- [ ] _Replace this line with individual tasks unique to your PR_

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: modules/appeals_api/README.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ paths:
       summary: Retrieve appeals status for the Veteran with the supplied ssn
       operationId: getAppealStatus
       security:
-        - api_key: []
+        - apikey: []
```
Comment (johnpaulashenfelter) [inline modules/appeals_api/README.yml]: Did the name change? Or was this bad documentation?

File: modules/claims_api/app/swagger/claims_api/claims_v0_controller_swagger.rb [hunk 2]
```diff
@@ -17,14 +20,7 @@ class ClaimsV0ControllerSwagger
...
           key :description, 'The ID of the claim being requested'
-          key :required, true
-          key :type, :string
-        end
-
-        parameter do
-          key :name, 'apikey'
-          key :in, :header
-          key :description, 'API Key given to access data'
+          key :example, ''
```
Comment (johnpaulashenfelter) [inline modules/claims_api/app/swagger/claims_api/claims_v0_controller_swagger.rb]: All of these examples are great 👍

Comment (ATeal) [reply modules/claims_api/app/swagger/claims_api/claims_v0_controller_swagger.rb]: So we're going to make a separate issue for adding examples to all of them, for now I'll remove the blank ones.

I started to do it, but it was pretty daunting

File: modules/appeals_api/README.yml (hunk 1 above)
Comment (ATeal) [reply modules/appeals_api/README.yml]: So it's being changed to align with the security schema. The other PR that's tied to this is generating curls from swagger, which requires properly formatted swagger.

**Pull request merged**